.prod/uwsgi.ini

```diff
@@ -9,3 +9,8 @@ gid = appuser
 master = 1
 processes = 2
 threads = 2
+cron = -1 -1 -1 -1 -1 /app/manage.py send_mail
+cron = 0 -1 -1 -1 -1 /app/manage.py retry_deferred
+cron = 20 -1 -1 -1 -1 /app/manage.py retry_deferred
+cron = 40 -1 -1 -1 -1 /app/manage.py retry_deferred
+cron = 0 0 -1 -1 -1 /app/manage.py purge_mail_log 7
```

Comment:
this is missing newline at the end too 😬 you had too empty lines before, now it's 0 empty lines, but it should be one

Reply:
Not sure what's going on with my IDE and github though -__-  I only see one empty line before so tried to delete one